modules/cold-extraction/ColdDataRetriever.py

```diff
@@ -89,7 +99,7 @@ def initialize_config_and_execute(valuesDict):
     else:
         cfind_add = ' -x stid.csv.xsl '
         out_folder = '.'
-
```

Comment:
Can you please make sure to remove all the line break changes in the pull request, and submit a new pull request? The changes displayed in the diff should be minimal and free from such line break changes and new line changes.

Otherwise, when we eventually merge the pull request, it will create an unnecessary merge conflict.

Reply:
yeah! I am sorry for doing it again.